airflow/providers/openlineage/extractors/base.py

```diff
@@ -78,7 +78,9 @@ def _is_operator_disabled(self) -> bool:
         return fully_qualified_class_name in self.disabled_operators
 
     def validate(self):
-        assert self.operator.task_type in self.get_operator_classnames()
+        if self.operator.task_type not in self.get_operator_classnames():
+            msg = f"Operator Type {self.operator.task_type} is not supported."
+            raise AssertionError(msg)
```

Comment:
I can't find usage of this method 🤔 so maybe here better to use some other error?

Reply:
@mobuchowski ?

Reply:
Let's remove it, must have been deleted earlier.

Reply:
Good idea, I've can't found the usage of it since it implemented, so we pretty safe to remove it